pages/linux/bpftrace.md

```diff
@@ -0,0 +1,28 @@
+# bpftrace
+
+> High-level tracing language for Linux eBPF.
+> More information: <https://github.com/iovisor/bpftrace>.
+
+- Display bpftrace version:
+
+`bpftrace -V`
+
+- List all available probes:
+
+`sudo bpftrace -l`
+
+- Run a one-liner program (e.g syscall count by program):
+
+`sudo bpftrace -e {{'tracepoint:raw_syscalls:sys_enter { @[comm] = count(); }'}}`
+
+- Run a program from a file:
+
+`sudo bpftrace {{path/to/file}}`
+
+- Trace a program by PID:
+
+`sudo bpftrace -e {{'tracepoint:raw_syscalls:sys_enter /pid == 123/ { @[comm] = count(); }'}}`
+
+- Do a dry run and display the output in eBPF format:
+
+`sudo bpftrace -d -e {{one_line_program}}`
```

Comment:
Just add the quotes to the argument because need it.
```suggestion
`sudo bpftrace -d -e '{{one_line_program}}'`
```

Reply:
I added inside of `{{' ... '}}`, just like the other examples on this file

Reply:
We had a discuss this in #4363 and the quotes should be outside `{{}}`. Could you please update the 3 example descriptions?

Reply:
@andrealmeid We had a discuss about this in #4363 and the quotes should be outside `'{{...}}'`. Could you please update the 3 example descriptions? And please *DO NOT* push force, is harder to us to track changes.

Reply:
@navarroaxel ok, sorry, I won't push force again